src/shared/filesystemUtilities.ts

```diff
@@ -27,13 +26,6 @@ export async function fileExists(filePath: string): Promise<boolean> {
     return true
 }
 
-export const readDirAsString = async (
-    pathLike: PathLike,
-    options: { encoding: BufferEncoding } = { encoding: DEFAULT_ENCODING }
```

Comment:
Is there any change that readdir wouldn't encode as utf8? Looks like that's the main reason why this function existed.

Reply:
libuv typically converts filesystem results to utf8, even on Windows, so I would be surprised if readdir returns something else.

Reply:
By default all the filesystem utils use utf8

Reply:
If that's the case, remove `DEFAULT_ENCODING` and `readFileAsString`.